src/components/MapPanel.vue

```diff
@@ -425,7 +432,11 @@
             topicLayers = component.options.topicLayers;
           }
         }
-        return topicLayers;
+        let topicLayersKeys = [];
+        for (let topicLayer of topicLayers) {
+          topicLayersKeys.push(topicLayer.title)
+        }
```

Comment:
I think we could rewrite this (lines 435 - 438) as something like: 
```
topicLayerKeys = topicLayers.map(topicLayer =>  topicLayer.title);
```

Reply:
thanks.  Similarly, I'm actually looking up .filter() and reminding myself how it works right now for the work I'm doing to make the checkbox ordering work.  I knew about .map(), but have never made it my habit to use it yet